docs/program/extend/modular-resources/_index.md

```diff
@@ -25,9 +26,8 @@ Two key concepts exist across all Viam resources (both built-in and modular) to
 
 ### APIs
 
-Every Viam [resource](/appendix/glossary/#term-resource) exposes an [Application Programming Interface (API)](https://en.wikipedia.org/wiki/API).
-This can also be understood as the protocol that the resource "speaks".
-Each API is described through <a href="https://developers.google.com/protocol-buffers" target="_blank">protocol buffers</a>.
+Every Viam {{< glossary_tooltip term_id="resource" text="resource" >}} exposes an [Application Programming Interface (API)](https://en.wikipedia.org/wiki/API), which allows you to interact with or "speak" to the resource.
```

Comment:
```suggestion
Every Viam {{< glossary_tooltip term_id="resource" text="resource" >}} exposes an [Application Programming Interface (API)](https://en.wikipedia.org/wiki/API).
This can be understood as a description of how you can interact with that resource.
```